policyengine_us/variables/gov/ed/pell_grant/efc/head/pell_grant_head_income.py

```diff
@@ -0,0 +1,8 @@
+from policyengine_us.model_api import *
+
+
+class pell_grant_head_income(Variable):
```

Comment:
this excludes spouse income?

Reply:
This does not exclude spouse income. Is there a better name for this that would make it more clear?

Reply:
how does the document describe it?

Reply:
The document calls it parent income for formula A and student/spouse income for formulas B and C.

Reply:
how about `pell_grant_primary_income`?

per chatgpt: https://chat.openai.com/share/6d013c41-719c-483c-9cf5-b684220303d4